br.com.eduk.app.yml

```diff
@@ -0,0 +1,62 @@
+app-id: br.com.eduk.app
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
```

Comment:
you probably need `--device=dri` since it's Electron based

Reply:
Fix